spec/controllers/sessions_controller_spec.rb

```diff
@@ -161,88 +195,132 @@
   end
 
   describe "POST confirm_email" do
-    before do
-      session[:username] = "coolcat423"
-      session[:provider] = "github"
-      session[:uid] = "12345"
-    end
+    describe "with Github authentication" do
```

Comment:
Similar to above -- here through line 288, I just indented the existing test cases into a describe "with Github authentication" do ... end block, and improved the assertions in the `"creates user and makes applicant"` test case.